src/application/components/SyntaxHighlighter.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import Highlight, { 
+  defaultProps,
+  Language,
+  PrismTheme,
+} from "prism-react-renderer";
+import { colors } from '@apollo/space-kit/colors';
+
+interface SyntaxHighlighterProps {
+  className?: string;
+  language: Language;
+  code: string;
+}
+
+const theme: PrismTheme = {
+  plain: {
+    color: 'currentColor',
+    backgroundColor: 'transparent',
+  },
+  styles: [
+    {
+      types: ['important', 'bold'],
+      style: {
+        fontWeight: 'bold'
+      }
+    },
+    {
+      types: ['italic'],
+      style: {
+        fontStyle: 'italic'
+      }
+    },
+    {
+      types: ['entity'],
+      style: {
+        cursor: 'help'
+      }
+    },
+    {
+      types: ['selector', 'attr-name', 'char', 'builtin', 'inserted'],
+      style: {
+        color: colors.teal.light,
+      }
+    },
+    {
+      types: ['comment', 'prolog', 'doctype', 'cdata'],
+      style: {
+        color: colors.orange.light,
+        fontWeight: 'bold'
+      }
+    },
+    {
+      types: ['punctuation'],
+      style: {
+        color: colors.grey.light,
+      }
+    },
+    {
+      types: [
+        'property',
+        'tag',
+        'boolean',
+        'number',
+        'constant',
+        'symbol',
+        'deleted',
+        'class-name',
+        'function'
+      ],
+      style: {
+        color: colors.pink.light,
+      }
+    },
+    {
+      types: ['atrule', 'attr-value', 'keyword'],
+      style: {
+        color: 'inherit',
+        background: 'transparent'
+      }
+    },
+    {
+      types: ['atrule', 'attr-value', 'keyword'],
+      style: {
+        color: colors.indigo.light,
+      }
+    },
+    {
+      types: ['regex', 'important', 'variable', 'description'],
+      style: {
+        color: colors.yellow.light,
+      }
+    }
+  ]
+};
+
+const SyntaxHighlighter = ({ 
+  code,
+  className: outerClassName,
+  language,
+}: SyntaxHighlighterProps) => {
+  return (
+    <Highlight
+      {...defaultProps}
+      language={language} 
+      theme={theme}
+      code={code}
+    >
+      {({ className, style, tokens, getLineProps, getTokenProps }) => {
+        return (
+          <pre 
+            className={[className, outerClassName].filter(Boolean).join(' ')} 
+            style={style}
+          >
+            <code>
```

Comment:
Minor nitpick: maybe add the className & style from above directly on the `code` (with `white-space: pre;` and `word-break: keep-all;`) and omit the `pre` element?

Reply:
`<pre><code>` seems to be the [recommended way](https://www.codiga.io/blog/display-code-snippets-in-html/) to represent a block of code in HTML. Would you be ok leaving this as is?

Reply:
I don't have any strong feelings here :)